GitTfs/Core/GitRepository.cs

```diff
@@ -697,7 +697,7 @@ public bool Checkout(string commitish)
         {
             try
             {
-                _repository.Checkout(commitish);
+                LibGit2Sharp.Commands.Checkout(_repository, commitish);
```

Comment:
According to the docs, the `Checkout` method is deprecated and `LibGit2Sharp.Commands.Checkout(...)` is the replacement.